Source/OxyPlot.GtkSharp/OxyPlot.GtkSharp.csproj

```diff
@@ -12,7 +12,7 @@
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <ProductVersion>10.0.0</ProductVersion>
+    <ProductVersion>8.0.30703</ProductVersion>
```

Comment:
project saved in MonoDevelop? The other changes are also just reordering, right?